10.Mender-server/docs.md

```diff
@@ -2178,6 +2178,161 @@ New changes in integration since 3.3.0:
 * Upgrade workflows to 2.3.0.
 
 
+## Mender 3.3.2
+
+_Released 03.10.2023_
+
+### Statistics
+
+A total of 2694 lines added, 1426 removed (delta 1268)
+
+| Developers with the most changesets | |
+|---|---|
+| Lluis Campos | 21 (32.3%) |
+| Krzysztof Jaskiewicz | 16 (24.6%) |
+| Alf-Rune Siqveland | 12 (18.5%) |
+| Peter Grzybowski | 8 (12.3%) |
+| Fabio Tranchitella | 4 (6.2%) |
+| Kristian Amlie | 2 (3.1%) |
+| Maciej Tomczuk | 2 (3.1%) |
+
+| Developers with the most changed lines | |
+|---|---|
+| Peter Grzybowski | 1309 (38.7%) |
+| Alf-Rune Siqveland | 740 (21.9%) |
+| Krzysztof Jaskiewicz | 730 (21.6%) |
+| Lluis Campos | 298 (8.8%) |
+| Maciej Tomczuk | 192 (5.7%) |
+| Kristian Amlie | 57 (1.7%) |
+| Fabio Tranchitella | 54 (1.6%) |
+
+| Developers with the most lines removed | |
+|---|---|
+| Peter Grzybowski | 26 (1.8%) |
+
+| Developers with the most signoffs (total 2) | |
+|---|---|
+| Kristian Amlie | 1 (50.0%) |
+| Lluis Campos | 1 (50.0%) |
+
+| Top changeset contributors by employer | |
+|---|---|
+| Northern.tech | 65 (100.0%) |
+
+| Top lines changed by employer | |
+|---|---|
+| Northern.tech | 3380 (100.0%) |
+
+| Employers with the most signoffs (total 2) | |
+|---|---|
+| Northern.tech | 2 (100.0%) |
+
+| Employers with the most hackers (total 7) | |
+|---|---|
+| Northern.tech | 7 (100.0%) |
+
+### Changelogs
+
+#### deployments (4.2.2)
+
+New changes in deployments since 4.2.1:
+
+##### Features
+
+* omit configuration values in the deployment details
+
+#### deployments-enterprise (4.2.2)
+
+New changes in deployments-enterprise since 4.2.1:
+
+##### Features
+
+* omit configuration values in the deployment details
+
+#### integration (3.3.2)
+
+New changes in integration since 3.3.1:
+
+* test: adjust integration tests after backporting RBAC fixes
+* Upgrade deployments-enterprise to 4.2.2.
+* Upgrade deployments to 4.2.2.
+* Upgrade integration to 3.3.2.
+* Upgrade iot-manager to 1.0.4.
+* Upgrade mender-cli to 1.8.2.
+* Upgrade mender-convert to 3.0.2.
+* Upgrade mender to 3.3.2.
+* Upgrade useradm-enterprise to 1.18.2.
+* Upgrade useradm to 1.18.2.
+
+#### iot-manager (1.0.4)
+
+New changes in iot-manager since 1.0.3:
+
+##### Bug Fixes
+
+* wrong capitalization for `Type`
```

Comment:
Doesn't make any sense to me :p

Reply:
Is it a doc change..? The author will have to stand for this.

Reply:
haha, yup!